server/tools/hatohol-db-initiator.in

```diff
@@ -132,6 +132,11 @@ def parse_default_conf():
 
 def get_default_conf_item(item_name):
     config = parse_default_conf()
+    if config is None:
+       global not_found_config_file
+       not_found_config_file = True
+       return "<Unknown>"
```

Comment:
hatohol_db_user & hatohol_db_password become "<Unknown>" if "--hatohol-db-user" and "--hatohol-db-password" aren't specified.
Is it expected behaviour?

```
$ hatohol-db-initiator --db_name hatoholtest --db_user "root" --db_password "hoge"  -c
Traceback (most recent call last):
  File "/usr/local/bin/hatohol-db-initiator", line 218, in <module>
    start(args)
  File "/usr/local/bin/hatohol-db-initiator", line 153, in start
    create_db_if_needed(cursor, args)
  File "/usr/local/bin/hatohol-db-initiator", line 44, in create_db_if_needed
    % (args.db_name, args.hatohol_db_user, args.hatohol_db_password))
  File "/usr/local/lib/python2.7/dist-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/usr/local/lib/python2.7/dist-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
_mysql_exceptions.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '<Unknown>@localhost IDENTIFIED BY '<Unknown>'' at line 1")
```